### ISSUE TITLE
[Feature Request] Show queue status in the tab title

### ISSUE THREAD

**Issue by rickles42** [comfyanonymous/ComfyUI#1040]
It would be handy and improve glanceability if ComfyUI showed something about the queue's current status in the tab/title bar. For example, simply showing the current length of the queue (it's shown as "Queue size: X" in the menu right now) would make it easy to tell whether I should pop back into ComfyUI and add more tasks to the queue.

Ideally the tab could display something like a completion ETA, but something as straightforward as the current queue length would still be very useful for multitaskers.

**Comment by Chubbly** [comment]
I'm not sure if it was added in response to this but the latest update of ComfyUI has the number of items in the queue displayed in the tab

![image](https://github.com/comfyanonymous/ComfyUI/assets/5073911/d475252d-14be-4230-bb30-99a60eee32ec)

**Comment by Chubbly** [comment]
> Hm... I'm running the latest ComfyUI and I'm not seeing that. For what it's worth, I'm using Edge, not Chrome. Either there's some issue with Edge support (seems unlikely for something so simple), or you've enabled an option I can't find, or you're seeing that status due to an extension of some sort that you've installed.

You are correct, it was https://github.com/pythongosssss/ComfyUI-Custom-Scripts that added it. Sorry for leading you astray!
